README.adoc

```diff
@@ -331,6 +331,60 @@ spec:
 ...
 ----
 
+== Services API Documentation
+
+Kiali can display API Documentation of your services. See https://user-images.githubusercontent.com/1235410/61569413-0e870480-aa3c-11e9-9527-bbea18eff475.png[Api documentation screen capture] and https://user-images.githubusercontent.com/1235410/61569419-1777d600-aa3c-11e9-82cc-f9510e5b9808.png[Api type list screen capture].
+
+=== Configure your services
+
+Your services must be annotated with the type of api ('rest', 'grpc', 'graphql') and a url to the spec of the api. 
+If the api spec is served from the service itself, Kiali will infer the hostname and port :
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: myservice
+  annotations:
+    kiali.kubernetes.io/api-type: rest
+    kiali.kubernetes.io/api-spec: /v1/api-spec  
+spec:
+...
+----
+
+The api spec can also be served from any http/s url, internal or external to the cluster :
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: petstore
+  annotations:
+    kiali.kubernetes.io/api-type: rest
+    kiali.kubernetes.io/api-spec: https://petstore.swagger.io/v2/swagger.json  
+spec:
+...
+----
+
+For now, only REST apis have their spec displayed but we are working to support gRpc and GraphQL soon.
+A live console to test your APIs directly with Kiali is also being worked on.
+
+=== Customize api docs annotations
+
+You can configure Kiali to use your own annotation names with the operator cr
```

Comment:
operator cr => Kiali Operator CR

Reply:
We call this the `Kiali CR` in all other places where we reference it. So I would continue to use that name -> Kiali CR

Reply:
Fixed